framework/web/User.php

```diff
@@ -165,6 +165,9 @@ public function init()
         if ($this->enableAutoLogin && !isset($this->identityCookie['name'])) {
             throw new InvalidConfigException('User::identityCookie must contain the "name" element.');
         }
+        if (!empty($this->accessChecker) && is_string($this->accessChecker)) {
+            $this->accessChecker = new $this->accessChecker;
```

Comment:
I think it's better to use `Yii::createObject()` here.

Reply:
Changed